### PR TITLE
LPD-28188

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPD-28188](https://liferay.atlassian.net/browse/LPD-28188)

### CODE REVIEW

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java [hunk 1]
```diff
@@ -444,6 +444,60 @@ public void testDeleteCustomObjectEntry() throws Exception {
...
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		_objectRelationships.add(objectRelationship);
+
+		_objectEntry1 = ObjectEntryTestUtil.addObjectEntry(
+			_objectDefinition1, _OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1);
+
+		_objectEntry2 = ObjectEntryTestUtil.addObjectEntry(
+			_objectDefinition2, _OBJECT_FIELD_NAME_2, _OBJECT_FIELD_VALUE_2);
+
```
Comment (marcelabc) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java:461]: This variable still exists from the previous test scenario. We don't need to create another and override this variable.

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by vngoliveira** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by vngoliveira** [comment]
ci:foward

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**